.npmignore

```diff
@@ -0,0 +1,3 @@
+components/
```

Comment:
What are your feelings on leaving `components` *in* the npm directory? The source is pretty small, and to me an advantage is that you can go digging in `node_modules` and find the original source instead of just build files—which I actually do surprisingly often. Apologies that this was originally my suggestion, but upon thinking further, I've heard complaints of unnecessarily `.npmignored` files making things difficult, particularly for people who may work offline or with a sketchy connection.